xwiki-commons-core/xwiki-commons-properties/src/main/java/org/xwiki/properties/PropertyDescriptor.java

```diff
@@ -118,4 +118,12 @@ default boolean isAdvanced() {
     default PropertyGroupDescriptor getGroupDescriptor() {
         return new PropertyGroupDescriptor(Collections.emptyList());
     }
+
+    /**
+     * @return the name of the feature associated to the parameter.
+     * @since 10.11RC1
+     */
+    default String getFeature() {
```

Comment:
For me the whole point was to make this a property group thing so located in the PropertyGroupDescriptor.

Reply:
Sure, good idea.

Reply:
I think it also mean that the properties you share the same instance of PropertyGroupDescriptor, otherwise you could end up with properties with the same group but different feature.

Reply:
@tmortagne  What happen if a field is annotate with `@PropertyFeature` and not `@PropertyGroup`? Should the group in the `PropertyGroupDescriptor` be null, or maybe an empty list? WDYT?

Reply:
I would go for null I think, makes more sense to me.